components/SpotDetails.js

```diff
@@ -0,0 +1,64 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function SpotDetails({ spot }) {
+  return (
+    <>
+      <DetailsSection>
+        <DetailsHeadline>Details</DetailsHeadline>
+        <h4>Adresse:</h4>
```

Comment:
Headlines follow a successive order from `h1` to `h2` to `h3` and so on. Did you make sure that you already have an `h2` headline in your overall html markup? Don't skip any sub-headlines; it's vital for semantics which in turn is important for SEO.